core/src/datayoga_core/block.py

```diff
@@ -4,21 +4,19 @@
 import logging
 import os
 import sys
-from enum import Enum
+from abc import ABCMeta, abstractmethod
```

Comment:
what is this ABCMeta?

Reply:
https://docs.python.org/3/library/abc.html

This module provides the infrastructure for defining [abstract base classes](https://docs.python.org/3/glossary.html#term-abstract-base-class) (ABCs) in Python